src/um/winbase.rs

```diff
@@ -361,6 +361,115 @@ pub const MS_CTS_ON: DWORD = 0x0010;
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN10_RS2)
+    ProcThreadAttributeDesktopAppPolicy = 18,
+#endif
+} PROC_THREAD_ATTRIBUTE_NUM;
+#endif
+#define ProcThreadAttributeValue(Number, Thread, Input, Additive) \
+    (((Number) & PROC_THREAD_ATTRIBUTE_NUMBER) | \
+     ((Thread != FALSE) ? PROC_THREAD_ATTRIBUTE_THREAD : 0) | \
+     ((Input != FALSE) ? PROC_THREAD_ATTRIBUTE_INPUT : 0) | \
+     ((Additive != FALSE) ? PROC_THREAD_ATTRIBUTE_ADDITIVE : 0))
+#define PROC_THREAD_ATTRIBUTE_PARENT_PROCESS \
+    ProcThreadAttributeValue (ProcThreadAttributeParentProcess, FALSE, TRUE, FALSE)
+#define PROC_THREAD_ATTRIBUTE_HANDLE_LIST \
+    ProcThreadAttributeValue (ProcThreadAttributeHandleList, FALSE, TRUE, FALSE)
+#endif // (_WIN32_WINNT >= 0x0600)
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN7)
+#define PROC_THREAD_ATTRIBUTE_GROUP_AFFINITY \
+    ProcThreadAttributeValue (ProcThreadAttributeGroupAffinity, TRUE, TRUE, FALSE)
+#define PROC_THREAD_ATTRIBUTE_PREFERRED_NODE \
+    ProcThreadAttributeValue (ProcThreadAttributePreferredNode, FALSE, TRUE, FALSE)
+#define PROC_THREAD_ATTRIBUTE_IDEAL_PROCESSOR \
+    ProcThreadAttributeValue (ProcThreadAttributeIdealProcessor, TRUE, TRUE, FALSE)
+#define PROC_THREAD_ATTRIBUTE_UMS_THREAD \
+    ProcThreadAttributeValue (ProcThreadAttributeUmsThread, TRUE, TRUE, FALSE)
+#define PROC_THREAD_ATTRIBUTE_MITIGATION_POLICY \
+    ProcThreadAttributeValue (ProcThreadAttributeMitigationPolicy, FALSE, TRUE, FALSE)
+#endif
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8)
+#define PROC_THREAD_ATTRIBUTE_SECURITY_CAPABILITIES \
+    ProcThreadAttributeValue (ProcThreadAttributeSecurityCapabilities, FALSE, TRUE, FALSE)
+#endif
+#define PROC_THREAD_ATTRIBUTE_PROTECTION_LEVEL \
+    ProcThreadAttributeValue (ProcThreadAttributeProtectionLevel, FALSE, TRUE, FALSE)
+#if (_WIN32_WINNT >= _WIN32_WINNT_WINBLUE)
+#endif
+*/
+pub const PROC_THREAD_ATTRIBUTE_NUMBER: DWORD = 0x0000FFFF;
+pub const PROC_THREAD_ATTRIBUTE_THREAD: DWORD = 0x00010000;
+pub const PROC_THREAD_ATTRIBUTE_INPUT: DWORD = 0x00020000;
+pub const PROC_THREAD_ATTRIBUTE_ADDITIVE: DWORD = 0x00040000;
+pub const ProcThreadAttributeParentProcess: DWORD = 0;
+pub const ProcThreadAttributeHandleList: DWORD = 2;
+pub const ProcThreadAttributeGroupAffinity: DWORD = 3;
+pub const ProcThreadAttributePreferredNode: DWORD = 4;
+pub const ProcThreadAttributeIdealProcessor: DWORD = 5;
+pub const ProcThreadAttributeUmsThread: DWORD = 6;
+pub const ProcThreadAttributeMitigationPolicy: DWORD = 7;
+pub const ProcThreadAttributeSecurityCapabilities: DWORD = 9;
+pub const ProcThreadAttributeProtectionLevel: DWORD = 11;
+pub const ProcThreadAttributeJobList: DWORD = 13;
+pub const ProcThreadAttributeChildProcessPolicy: DWORD = 14;
+pub const ProcThreadAttributeAllApplicationPackagesPolicy: DWORD = 15;
+pub const ProcThreadAttributeWin32kFilter: DWORD = 16;
+pub const ProcThreadAttributeSafeOpenPromptOriginClaim: DWORD = 17;
+pub const ProcThreadAttributeDesktopAppPolicy: DWORD = 18;
```

Comment:
I *think* you can use the ENUM! macro for these, and explicitly set the values like [this](https://github.com/retep998/winapi-rs/blob/a6dd5510212a2f0f5aae66e60949bd6fdaa0a44d/src/um/d2d1effects.rs#L153)

I'm not sure if that's actually better since if you do this you'll need to cast it when used in the macro